android/app/build.gradle

```diff
@@ -154,8 +154,8 @@ android {
         testInstrumentationRunner 'androidx.test.runner.AndroidJUnitRunner'
 
 
-        versionCode 32
-        versionName "1.2.9"
+        versionCode 34
```

Comment:
this versionCode shouldn't be 33??  because the previous one was 32, the same with versionName or it's been took automatically by android studio??

Reply:
The version code and version name has to be updated by 1 each time. If you read the note i left on the PR description, it explains why it jumps 2 numbers.

Reply:
got it